### PR TITLE
Allow unfreezing features in staged network dict

### PR DESCRIPTION
**Pull request by Max-Ryujin**
- added the ability to unfreeze features 
- added experiments that include 
  - starting with features from partition epoch 16 and freezing them for the first 16 partition epochs
  - freezing features after partition epoch 256 until the end of the training

### CODE REVIEW

File: users/vieting/experiments/switchboard/ctc/feat/experiments.py
Comment (vieting) [inline users/vieting/experiments/switchboard/ctc/feat/experiments.py:835]: You should never use absolute paths as strings in a setup like this. The clean solution would be to get the checkpoint from a previous training. In case this is just experimental for you to test this, I'd rather wait for you to check the results and only clean and merge this if it has shown to be helpful.

File: users/vieting/experiments/switchboard/ctc/feat/experiments.py
Comment (vieting) [inline users/vieting/experiments/switchboard/ctc/feat/experiments.py:857]: If this is always the same, please declare a variable and re-use that.

**Review by vieting — CHANGES_REQUESTED** [review]

File: users/vieting/experiments/switchboard/ctc/feat/experiments.py
Comment (Max-Ryujin) [reply users/vieting/experiments/switchboard/ctc/feat/experiments.py:843]: It is only possible to ignore certain variables. As far as I know this is the easiest way to do this

**Comment by Max-Ryujin** [comment]
Since the freezing experiments did not show any improvement, I will remove most of these changes and only add the ability to unfreeze features

**Review by vieting — APPROVED** [review]

**Pull request merged**
